packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -0,0 +1,64 @@
+import { attr } from '@microsoft/fast-element';
+import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import { template } from '../theme-provider/template';
+import { styles } from './styles';
+import {
+    WaferMapColorBy,
+    WaferMapColorsScale,
+    WaferMapDataType,
+    WaferMapDie,
+    WaferMapOrientation,
+    WaferMapQuadrant
+} from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-wafer-map': WaferMap;
+    }
+}
+
+/**
+ * A nimble-styled WaferMap
+ */
+export class WaferMap extends FoundationElement {
```

Comment:
We should add a story to [the Nimble Storybook documentation](https://nimble.ni.dev/storybook/?path=/docs/button--outline-button) for this component. It should contain:
1. the component
2. controls that allow users to set each of these public properties. Each control should have documentation (can be copied from the spec)
3. a banner mentioning that the component is under development and not ready for use (similar to the banner we have [for the tooltip](https://nimble.ni.dev/storybook/?path=/docs/tooltip--tooltip))

The easiest way to add a story is to copy the `*.stories.ts` file from another component and modify it for this one.

Eventually you will also create a `*-matrix.stories.ts` file, which renders the component in every possible configuration and is used for visual comparison auto testing.

I am ok with this happening in a follow-up PR since the component doesn't render anything yet. Just want to make sure you're aware of this step. I think we'll want this as soon as the component starts rendering content.

Reply:
Yes, we have a task for it (https://dev.azure.com/ni/DevCentral/_workitems/edit/2208925). We started to work on it, but it will be in a separate PR, after this PR will be in.

Reply:
Thank you for information, we will used them in the next work. :)
fyi @DStavilaNI if this will help you in the Storybook task